src/terrain/generation/land_generator.cpp

```diff
@@ -137,86 +147,92 @@ JsonToTile::read_elements_can_stamp(const Json::Value& data) {
     return out;
 }
 
-FromRadius::FromRadius(const Json::Value& data) : JsonToTile(data) {
-    number_ = data["Radius"]["number"].asInt();
-    radius_ = data["Radius"]["radius"].asInt();
-
-    center_variance_ = data["DC"].asInt();
-}
+FromRadius::FromRadius(const Json::Value& data) :
+    JsonToTile(data), radius_(data["Radius"]["radius"].asInt()),
+    number_(data["Radius"]["number"].asInt()), center_variance_(data["DC"].asInt()) {}
 
 TileStamp
-FromRadius::operator()(int current_sub_region) const {
+FromRadius::get_this_stamp(ssize_t current_sub_region) const {
```

Comment:
Why `ssize_t` and not just `size_t`? `ssize_t` isn't defined on all platforms.

If you want to use it, then add this to `CMakeLists.txt`:
```cmake
check_type_size("ssize_t", SSIZE_T_SIZE)
if(HAVE_SSIZE_T_SIZE)
    target_compile_definitions(FunGame PUBLIC FUNGAME_HAVE_SSIZE_T)
endif()
```

And then to `types.hpp`:
```c++
#ifndef FUNGAME_HAVE_SSIZE_T
#  if sizeof(size_t) == 4
using ssize_t = int32_t;
#  elif sizeof(size_t) == 8
using ssize_t = int64_t;
#  else
#    error Unknown platform, cannot define ssize_t.
#  endif
#endif

Reply:
I should really explain this as it also bothered me. The json library uses signed ints to index into json lists. For this reason region and sub region are both saved and returned as signed ints.

Reply:
Well then add the code I linked

Reply:
Well I looked into it and it seemes I was wrong. I don't think the index needs to be unsigned and it dosen't see to be even used as a json index.